build_debian.sh

```diff
@@ -342,6 +347,10 @@ if [ "${enable_organization_extensions}" = "y" ]; then
    fi
 fi
 
+## Remove gcc and python dev pkgs
+sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y remove libpython2.7-dev
+sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y remove gcc
```

Comment:
Unless you have a reason to keep them separate, you can combine lines 351 and 352 into one command, e.g.:

`sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y remove gcc libpython2.7-dev`

Reply:
Sure will do that.